sql/core/src/test/scala/org/apache/spark/sql/execution/columnar/CachedBatchSerializerSuite.scala

```diff
@@ -120,6 +121,11 @@ class CachedBatchSerializerSuite  extends QueryTest with SharedSparkSession {
       classOf[TestSingleIntColumnarCachedBatchSerializer].getName)
   }
 
+  protected override def afterAll(): Unit = {
```

Comment:
let's do it in `beforeAll` as well, to guarantee that we are testing a fresh serializer in this test suite.

Reply:
Sure!

Reply:
Oh. It should be here. `beforeAll` will leave the leftover and will break the other suite.

Reply:
Do you want to have both beforeAll and afterAll?

Reply:
Got it~ I missed `as well` in your comment.